test/arrayDifference.test.js

```diff
@@ -0,0 +1,9 @@
+const { arrayDifference } = require("../lib");
+
+test("arrayDifference([2, 1], [2, 3]) should return [1]", () => {
+  expect(arrayDifference([1, 2], [2, 3])).toEqual([1]);
+});
+
+test("arrayDifference([2, 3, 1, 4], [2, 3]) should return [3, 1]", () => {
```

Comment:
Is this an error or am I only not understanding the function? Shouldn't this return [1, 4] ?